src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorCollectRunner.CollectionRule.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using Microsoft.Diagnostics.Monitoring.TestCommon.Runners;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests.Runners
+{
+    partial class MonitorCollectRunner
+    {
+        private const int CollectionRuleCompletedEventId = 31;
+        private const int CollectionRuleFailedEventId = 30;
```

Comment:
I'm assuming this corresponds to what's in `LoggingExtensions`; I guess I don't see any reason why those numbers would necessarily change, but something still feels odd to me about leaving these here like this. Assuming there's no way to actually link them up to the actual values, maybe we leave a comment saying that these values need to correspond with what's in there? Unless you think it's obvious, but from a long-term maintainability standpoint I think that would help with readability.

Reply:
Maybe I'll add a separate class in dotnet-monitor that has the event IDs and then link that into this assembly so there is one source of truth.